src/components/KanbanCard/KanbanCard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const KanbanCard = (props) => {
+  const { title, priorityName, created_byFirstName, assigned_byFirstName } = props;
```

Comment:
In general for variable naming I would recommend sticking w/ all camel case and not a mix of underscores and casing:

`createdByFirstName`

`assignedByFirstName`